### PR TITLE
completed css-three-column-layout exercise, avallon smith

### PR DESCRIPTION
**Pull request by avallonsmith**
image

### CODE REVIEW

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  padding: 15px 10px;
+  border-radius: 100px;
+}
+
+p {
+  font-size: small;
+}
+
+.p-l {
+  padding-left: 250px;
```
Comment (Zhiwutian) [inline css-three-column-layout/styles.css:61]: you don't want to use padding for layout. make sure to avoid doing that. There is always a better way!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Make sure to remove the padding for layout, and let me know if you have questions on another approach!

**Comment by avallonsmith** [comment]
image
no more padding B)

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
make sure to delete the commented code!

**Comment by avallonsmith** [comment]
Should be good now!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**